src/components/FloatingFlow.vue

```diff
@@ -214,10 +221,31 @@ export default {
       this.bringDialogToFront(id);
     },
     resetApp: function(){
-      Object.assign(this.$data, initialState());
-      var closeItems = document.querySelectorAll('.mapboxgl-popup-close-button');
+      this.setState(initialState());
+    },
+    setState: function(state){
```

Comment:
Would it be cleaner to save all variables that are in a state in their own object? 

like as in:

```javascript
this.state = {
mainTabName: ...
z-index: ...
dockedArray: ...
}
```
Then we could do a single `Object.assign`
```javascript
Object.assign(this.state, state);
```